src/Screens/Main/Chat/Input.tsx

```diff
@@ -38,28 +44,46 @@ export default ({ buddyId }: Props) => {
     sendMessage({ buddyId, text: messageContent });
   };
 
+  const inputContainerBg = showPending ? colors.lightestGray : colors.white;
+
+  React.useEffect(() => {
+    if (isPending) {
+      let pendingTimeOut = setTimeout(() => setShowPending(true), 500);
+
+      return () => clearTimeout(pendingTimeOut);
+    } else {
+      setShowPending(false);
+    }
+  }, [isPending]);
```

Comment:
this looks soliid. to make it better, you could move setShow pending declaratoin just above this one. Proximity is golden with this kinda stuff.